### ISSUE TITLE
SSH connection doesn't work properly

### ISSUE THREAD

**Issue by cutesmilee** [alephsecurity/xnu-qemu-arm64-tools#19]
after signing all the things, and also generated an ssh key, and executing "/path/to/dropbear -r /var/dropbear_key" (the key i generated), if i try to connect to ssh with "ssh root@localhost -p 2222" it gives me: "Connection closed by 127.0.0.1 port 2222", idk a lot about networking, i'd like to know why could this happen, and eventually how to fix it.
thanks in advance!

**Comment by sickcodes** [comment]
Are you building on Linux or Mac?

**Comment by cutesmilee** [comment]
i only saw your reply now, anyway i'm on macOS Big Sur 11.2.1

**Comment by sickcodes** [comment]
Yeah this is the Linux version, you can follow this repo: https://github.com/alephsecurity/xnu-qemu-arm64-tools

Or specifically, build the TCP Tunnel on Mac and then replace it inside the disk image (mount it as per the `losetup` notes in readme).

**Comment by cutecodee** [comment]
"Thu Jan  1 00:13:35 1970 localhost com.apple.xpc.launchd[1] (TcpTunnel) <Notice>: Service only ran for 0 seconds. Pushing respawn out by 10 seconds."
now the problem is that for some reason tcptunnel is killed every time, but i signed it, it's weird

**Comment by cutecodee** [comment]
solved this, but i have the same problem as before, when i try to connect to the vm, it tries to connect, and after 10/20 seconds it says "Connection closed by 127.0.0.1 port 2222"

**Comment by cutesmilee** [comment]
still the same problem, seems like the tunnel works, and also dropbear, since i don't receive any error, but when i try to connect using ssh it closes the connection
